hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -230,6 +230,7 @@ export const PendingTables = () => {
     const toggleVisibility = () => dispatch(togglePendingTable());
     const cancelOrder = (orderId: number) => dispatch(cancelOrderThunk(orderId));
     const [showCancelOrderModal, setShowCancelOrderModal] = useState(false);
+    const [orderId, setorderId] = useState(0);
```

Comment:
Can you replace the `0` with `null`, because setting it to 0 indicates it's given an id, which by default it should not be.